lintreview/tools/ansible.py

```diff
@@ -1,3 +1,4 @@
+from __future__ import absolute_import
```

Comment:
Couldn't we move all the imports to be relative? Python 2.7 supports relative imports.